CLAUDE.md

```
@@ -0,0 +1,64 @@
+npm run lint               # ESLint check
+npm run fix                # Lint fix + Prettier format
+npm run preflight          # Full pre-merge check: format, lint, build, coverage (emulator)
+npm run dev                # Build + run server (stdio)
+npm run dev:http           # Build + run server (HTTP transport)
+```
+
+Single test file: `npx vitest run path/to/file.test.ts`
+
+## Architecture
+
+### Server Core (`src/index.ts`)
```

markdownlint-cli2 (0.22.0)

[warning] Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)